src/azul/__init__.py

```diff
@@ -552,32 +552,40 @@ def indexer_concurrency(self):
         return int(os.environ['AZUL_INDEXER_CONCURRENCY'])
 
     @property
-    def notify_queue_name(self):
-        return self.qualified_resource_name('notify')
+    def notifications_queue_name(self):
+        return self.qualified_resource_name('notifications')
 
     @property
-    def document_queue_name(self):
-        return config.qualified_resource_name('documents', suffix='.fifo')
+    def tallies_queue_name(self):
+        return config.qualified_resource_name('tallies', suffix='.fifo')
 
     @property
-    def fail_queue_name(self):
-        return config.qualified_resource_name('fail')
+    def tallies_retry_queue_name(self):
+        """
+        Any tallies that fail being processed in the tallies queue will be
+        retried with more RAM in the tallies.retry queue.
```

Comment:
> ... tallies.retry queue.

Was this a typo?

Reply:
Yes. Hannes preffered the . as a seperator if it was possible, but I tried it and it doesn't work with Terraform.